src/garvis/cli.py

```
@@ -257,6 +271,33 @@ async def _run(args: argparse.Namespace) -> int:
+                return 0
+
+            if approval.casefold() not in {"y", "yes", "n", "no"}:
+                print("GARVIS approval cancelled: enter Y or N.", file=sys.stderr)
+                return 2
+
+            try:
+                reply = await assistant.respond(
+                    approval,
+                    session_id=args.session,
+                )
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)